### DISCUSSION TITLE
How to get the api payload

### DISCUSSION THREAD

**Discussion by MSIH** [KeyserDSoze/Rystem.OpenAi#44] [Q&A]
how can i see the actual payload sent to the api? 

I want to store the info in a database, so I can reference in the future with another tool.

**Comment by KeyserDSoze** [comment]
Do you need the entire request? I have had the idea some weeks ago to add an ILogger interface to all the api to let this customization open through dependency injection. I would release it for next week. Can it solve your problem? Having the body in message in the ILogger.Log method?

**Reply by KeyserDSoze** [reply]
I'm  thinking to add the integration with ILogger. 
This is a preview of what I want to do
![image](https://github.com/KeyserDSoze/Rystem.OpenAi/assets/40713438/38d33759-4b10-46ae-bbd1-6ee81118e00f)
Here I can use ILogger interface to track everything with this classic log levels
![image](https://github.com/KeyserDSoze/Rystem.OpenAi/assets/40713438/1ee2fc87-87e0-4d20-936a-73e68df13087)
